gedcom/test_families.py

```diff
@@ -36,11 +62,13 @@ def tearDown(self):
     def test_default_init(self):
         """make sure the object is empty on init
         """
+        self.fam.families.clear()
```

Comment:
Not sure what this is for.  We have a setup and tear down which deletes and recreates the fam object already.